### ISSUE TITLE
Panel content is not clearly visible in 400% zoom mode

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [microsoft/fluentui#10905]
User Experience:
Users with low vision are getting affected as the elements in the panel are not clearly visible in 400% mode. Users will face difficulty in identifying the elements in the panel if they are not visible.

Repro Steps:
Step 1: Zoom the page to 400% zoom.
Step 2: Open Fabric UI URL in Edge browser.
Step 3: Navigate to Panel page and open any panel
Step 4: Verify whether all the panel and elements available in the panel are clearly visible in 400% zoom.

Actual:
Panel and all the elements are not Clearly visible in 400% zoom

Expected:
Panel and all the elements should be Clearly visible in 400% zoom

![image](https://user-images.githubusercontent.com/53210862/67095685-2ead9780-f1d4-11e9-9fb5-188c85ed8b8e.png)

**Comment by marygans** [comment]
Hi @SyedRehmanRazvi,  thanks for submitting this issue!

You mentioned the narrator's user experience on zoom. What is your expectation for that? It would be helpful as I am attempting to repro. 

I was also unable to repro the zooming issue. Do you mind sharing the details of the environment you are experiencing this in? 

Thanks!

**Comment by SyedRehmanRazvi** [comment]
Hi @marygans, Its a typo error issue is not for the screen reader user but for low vision users who zoom the page. We have updated the bug accordingly.

Thanks in Advance

**Comment by SyedRehmanRazvi** [comment]
Hi @marygans We have validated the issue in the below URL:
https://fabricweb.z5.web.core.windows.net/oufr/7.53.0/#/examples/panel
Below is the test environment and display settings details:
Windows OS Version: Version 1903 (OS Build 19002.1002)
Browser: Edge
Edge Browser Version: Microsoft Edge 44.19002.1.0
Display resolution : 1366*768
We have also verified in the below resolution:
Display resolution : 1280*768. Please find the below snapshot:
![image](https://user-images.githubusercontent.com/53210862/67183935-381b4780-f400-11e9-9a8f-eb530daca651.png)

**Comment by ecraig12345** [comment]
This issue is actually about Panel, not Dialog (I've updated the title and description to reflect that). @marygans You may need to use a smaller window size to repro this--more like small/medium laptop screen size.

**Comment by micahgodbolt** [comment]
changed priority as this is a wcag 2.1 requirement

https://www.w3.org/WAI/WCAG21/Understanding/reflow.html

**Comment by micahgodbolt** [comment]
@betrue-final-final  are you looking into the reflow requirements at all? Is there a design change that needs to happen for panel at smaller breakpoints?

**Comment by betrue-final-final** [comment]
The buttons should not be clipped, but otherwise the panel should be ok as long as it provides for a vertical only scroll. We do not have any control over content, so we can't predict when it would need a horizontal scroll, though our controls could constrain width to the window possibly. That is beyond Panel.

**Comment by xugao** [comment]
we no longer have a panel example on our doc page that has this issue. however, this is still related to #7625 and how user cannot scroll header/footer. closing this one as a duplicate

**Comment by ecraig12345** [comment]
Reopening since we will need to address 400% zoom issues at some point.

This currently repros on the "Panel - Footer" example on a smaller monitor (we can't assume everyone has giant monitors).
https://fabricweb.z5.web.core.windows.net/oufr/7.115.2/#/examples/panel

**Comment by mltejera** [comment]
For internal MSFT folks, [admin-controls](https://aka.ms/admincontrols) has done some work to solve for this:

Demo:
[https://aka.ms/admincontrols](http://aka.ms/adminexperiments/#/examples/DetailPanelV2)

Src:
[https://uifabric.visualstudio.com/iss/_git/m365-admin?path=%2Fpackages%2Fexperiments%2Fsrc%2Fcomponents%2FDetailPanelV2](https://uifabric.visualstudio.com/iss/_git/m365-admin?path=%2Fpackages%2Fexperiments%2Fsrc%2Fcomponents%2FDetailPanelV2)

**Comment by mltejera** [comment]
This issue definitely needs to be kept open to support WCAG 2.1 at some point.

**Comment by mltejera** [comment]
Update on this topic.

In our DPv2 implementation noted above, that wraps this panel, we put the header and footer into the main body are at short window heights. This pattern broke down when there were long lists in the body area, forcing the user to tediously scroll.

We now have a boolean flag to opt out, keeping the sticky footer behavior, but shrinking the vertical padding from 16->8px to get back a little room for the content.

**Comment by TristanWatanabe** [comment]
Fixed by #19547
